users_ldap_groups/__openerp__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "Groups assignment",
-    "version": "1.2",
+    "version": "7.0.1.2.1",
```

Comment:
@hbrunn This was discussed here: https://github.com/OCA/maintainer-tools/pull/100

@pedrobaeza I'm unsure about that but ok let's keep it quit in 7.0